Cows-HacknSlash/Assets/Scripts/NPC/RatBehaviour.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.AI;
+
+public class RatBehaviour : MonoBehaviour {
+
+   
```

Comment:
Double whiteline, will approve for now as i'm going to do another code fix iteration soon